adapters/grid/grid.go

```diff
@@ -394,8 +395,16 @@ func (a *GridAdapter) MakeBids(internalRequest *openrtb2.BidRequest, externalReq
 
 	for _, sb := range bidResp.SeatBid {
 		for i := range sb.Bid {
-			bidMeta, err := getBidMeta(sb.Bid[i].Ext)
+			bidMeta, err := getBidMeta(sb.Bid[i].Ext) //nolint: ineffassign,staticcheck // ineffectual assignment to err
+			// TODO(dmitris) - find out why uncommenting the error handling below causes unit tests to fail with:
+			//     test_json.go:130: gridtest/exemplary/multitype-banner.json: MakeBids had wrong error count. Expected 0, got 1 ([nil json.RawMessage passed to getBidMeta])
+			// if err != nil {
```

Comment:
Same comment as above, i think the code on like 401/402 should resolved. Either uncommented, or deleted.

Reply:
deleted in [befeec1](https://github.com/prebid/prebid-server/commit/befeec1073df522d998b2f1c1e118d4931671b3a)